src/QMCDrivers/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -1051,7 +1051,7 @@ bool QMCFixedSampleLinearOptimize::adaptive_three_shift_run()
     for (int j = 0; j < parameterDirections.size(); j++)
     {
       if (j != central_index)
-        parameterDirections.at(j).at(i + 1) -= parameterDirections.at(1).at(i + 1);
```

Comment:
Please remove the following commented line.

Reply:
Done